pkg/catalogv2/helm/helm3.go

```diff
@@ -28,6 +36,8 @@ func fromHelm3Data(data string, isNamespaced IsNamespaced) (*v1.ReleaseSpec, err
 	return fromHelm3ReleaseToRelease(release, isNamespaced)
 }
 
+// fromHelm3ReleaseToRelease receives a helm3 release.
```

Comment:
// fromHelm3ReleaseToRelease receives a helm3 release struct. Returns a pointer
// to a rancher v1.ReleaseSpec struct constructed from the helm3 release struct.